### PR TITLE
Add instructions to mentors on how to add ideas

### PR DESCRIPTION
**Pull request by dpshelio**
It also excludes the readme and template from being rendered by Jekyll

@eteq  @taldcroft @Cadair  - let me know if this readme looks ok

### CODE REVIEW

**Comment by Cadair** [comment]
I would maybe put a TLDR quick start at the top of the readme for the more experienced folks that just need a quick intro. Maybe a list of steps like "make dir, copy template, PR".

File: _projects/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+[chat platform we have](http://openastronomy.org/) (Thanks @Cadair!).
+The steps are set up from the point of view of I have an idea, what do I need so
+it's online - so the hardest steps are towards the bottom, and you read only up
+to the point you feel confident.
+
+*TLDR - Are you an expert?* Fork the repo and modify
+the [template file](./_template.md) according to your idea, save it under
+`_projects/YYYY/sub-org/` with a meaningful filename and do a Pull-request.
+
+## What to I need to have an idea?
```
Comment (eteq) [inline _projects/README.md]: ``to``->``do``?

File: _projects/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+approaches.
+
+## Format needed
+
+Cool,  you've got already all the above?
+
+All you need now is to put it in a plain text file following
+the [template file](./_template.md) (you may click in raw to see how it looks).
+
+The file is composed in two parts (separated by a `---`), the top is like a yaml
```
Comment (eteq) [inline _projects/README.md]: "like a"?  I'm not sure what you mean by that.  I'm also not sure it's that important for a mentor to know that?  Maybe just drop the part after the first comma in this sentence?

File: _projects/README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+file, the bottom is in markdown format. Change the text in the template keeping
+special care of the structure of the yaml file and the headings on the markdown
+part (_e.g._, the top level starts with a 4 level title: `####`).
+
+
+## How do I do so it appears online?
```
Comment (eteq) [inline _projects/README.md]: ``How``->``What``

File: _projects/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Are you familiar with git/github? If you are not contact us and share the above
+file with us, we will take care to put it in place.
```
Comment (eteq) [inline _projects/README.md]: Say who "us" is - give a specific email address or something like that.  Remember that if someone doesn't find the information necessary to submit in idea in this document, they probably won't submit anything... Ridiculously low barrier to entry is the goal!

File: _projects/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+If, in the other hand, you are feel confident with github: go ahead, fork this
+repository, create a subdirectory for your sub-organisation (if that doesn't
```
Comment (eteq) [inline _projects/README.md]: I think this needs to be very specific.  For example, "find the subdirectory for your sub-organisation and the year of the project (for example, if you are submitting an astropy project, go to /_projects/2017/astropy).  If your organization does not exist, create it following the form of one of the other organizations."

File: _projects/_template.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+Mauris imperdiet viverra turpis a porttitor. Duis laoreet leo sit amet odio
+suscipit aliquet. Etiam vel malesuada est, in molestie nibh. Sed efficitur
+pharetra lorem, sed tempor sapien commodo ac. Vivamus sollicitudin pretium erat.
+Pellentesque luctus vehicula tortor, nec volutpat turpis euismod vel. Sed neque
+ipsum, imperdiet nec massa sit amet, accumsan lobortis augue. Donec luctus massa
+vitae velit scelerisque volutpat ut nec ex. Sed in velit tincidunt, volutpat
+turpis sit amet, dignissim nisi. Donec a mattis eros. Pellentesque vitae leo
+quam.
+
+#### Milestones
```
Comment (eteq) [inline _projects/_template.md]: Add a note along the lines of "note that Milestones are *not* necessary - just leaving a description is sufficient if you don't have milestones."  When I looked at this I assumed the milestones were mandatory, and only by looking at the README did I realize it's not...

File: _projects/_template.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+desc: This is the first idea for astropy
+# add a description of your project
+requirements:
+ - The proposal has to contain a clear concept.
+ - patch on any astropy affiliated package.
+difficulty: low
+issues:
+ - https://github.com/astropy/astropy/issue/4444
+ - https://github.com/astropy/astropy/issue/5555
+mentors:
```
Comment (eteq) [inline _projects/_template.md]: This should be "possible mentors"?  We never know until the project starts, of course.

File: _projects/_template.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
+---
+name: Example Astropy
+desc: This is the first idea for astropy
+# add a description of your project
+requirements:
+ - The proposal has to contain a clear concept.
```
Comment (eteq) [inline _projects/_template.md]: I don't understand what this section means.  Requirements *for the student*?  Like "knows some astronomy"?  Or requirements for the project?

File: _projects/_template.md [hunk 9]
```diff
@@ -0,0 +1,50 @@
+---
+name: Example Astropy
+desc: This is the first idea for astropy
```
Comment (eteq) [inline _projects/_template.md]: What's the difference between this, and the "description" below?  I suggest removing this and having the name be the only "description" in this section

File: _projects/_template.md [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+difficulty: low
+issues:
+ - https://github.com/astropy/astropy/issue/4444
+ - https://github.com/astropy/astropy/issue/5555
+mentors:
+ - astrofrog
+initiatives:
+ - GSOC
+ - SOCIS
+tags:
```
Comment (eteq) [inline _projects/_template.md]: What does this mean?  I think without structure people will be confused by what they're supposed to do with it.  I suggest leaving it off.

File: _projects/_template.md [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+ - python
+collaborating_projects:
+ - astropy
```
Comment (eteq) [inline _projects/_template.md]: Again, why is this here?  Isn't the file going to already be *in* the astropy section?

**Comment by eteq** [comment]
Oh, and I didn't note it in the comments above, but I suggest there be just *one* template rather than one per project directory.  But in the project directory maybe we can place a README that points back to *this* README?

File: _projects/README.md (hunk 1 above)
Comment (dpshelio) [inline _projects/README.md]: thanks, done.

File: _projects/README.md (hunk 2 above)
Comment (dpshelio) [inline _projects/README.md]: the top uses yaml formatting, the rest markdown. But sure, now it's simpler.

File: _projects/README.md (hunk 3 above)
Comment (dpshelio) [inline _projects/README.md]: done

File: _projects/README.md (hunk 4 above)
Comment (dpshelio) [inline _projects/README.md]: email added.

File: _projects/README.md (hunk 5 above)
Comment (dpshelio) [inline _projects/README.md]: Changed, thanks!

File: _projects/_template.md (hunk 9 above)
Comment (dpshelio) [inline _projects/_template.md]: This is a one line description - it appears on the box together with the title. Added a description in the comment.

File: _projects/_template.md (hunk 8 above)
Comment (dpshelio) [inline _projects/_template.md]: For the student - Changed accordingly.

File: _projects/_template.md (hunk 7 above)
Comment (dpshelio) [inline _projects/_template.md]: But that should be the point of contact at the moment for the student writing the application.

File: _projects/_template.md (hunk 10 above)
Comment (dpshelio) [inline _projects/_template.md]: I've added a description,

File: _projects/_template.md (hunk 11 above)
Comment (dpshelio) [inline _projects/_template.md]: Not the directory structure is just to keep the things in order. This is to be able to separate them by sub-orgs and enables cross-collaborative project.

File: _projects/_template.md (hunk 6 above)
Comment (dpshelio) [inline _projects/_template.md]: done

**Comment by dpshelio** [comment]
> Oh, and I didn't note it in the comments above, but I suggest there be just one template rather than one per project directory. But in the project directory maybe we can place a README that points back to this file?

The "templates" on the projects are just so they appear on the website, so you can have an idea of how they look. They will be removed once we get a single real idea. Adding random files in different directories need to then be added to the exclude list in the config file.

> One other thing: maybe the _projects/README.md file should actually be /IDEAS.md? It's really not at all obvious that I should look in _projects to find this, because the leading underscore makes it seem like it's a data directory or something rather than a place to actually look for content.

The leading underscore is a jekyll thing (if I'm not wrong). We can move the `_projects/README.md` above, but I'm just worried that the top directory gets too full and this page is not only for GSOC.
